package.json

```diff
@@ -52,8 +52,10 @@
         "onCommand:isg-cnc.FindNextTFS",
         "onCommand:isg-cnc.RemoveAllBlocknumbers",
         "onCommand:isg-cnc.AddBlocknumbers",
+        "onCommand:isg-cnc.StartDocu",
         "onCommand:isg-cnc.Beautify",
-        "onCommand:isg-cnc.FindNonAsciiCharacters"
+        "onCommand:isg-cnc.FindNonAsciiCharacters",
+        "onCommand:cnc-show-filecontent"
```

Comment:
what does the command "cnc-show-filecontent"?

Reply:
It's a leftover from experimenting i think. I removed it, it isn't used anymore